Oddities.WinHelp/WinHelpFile.cs

```diff
@@ -9,7 +9,7 @@ namespace Oddities.WinHelp;
 /// <para>[2]: P. Davis and M. Wallace — Windows Undocumented File Formats.</para>
 /// </remarks>
 [PublicAPI]
-public struct WinHelpFile
+public record struct WinHelpFile
```

Comment:
Since this particular struct has private fields, it doesn't make much sense in converting it to a `record struct`. Let's leave it as it was before.